src/bin/lttng-sessiond/ust-metadata.c

```diff
@@ -300,8 +300,21 @@ int ust_metadata_enum_statedump(struct ust_registry_session *session,
 				goto end;
 			}
 		}
+		ret = lttng_metadata_printf(session, "\"");
+		if (ret) {
+			goto end;
+		}
+
+		if (entry->u.options.is_auto) {
+			ret = lttng_metadata_printf(session, ",\n");
+			if (ret) {
+				goto end;
+			}
+			continue;
```

Comment:
I don't like this "continue". We use if/else everywhere else in the code to make the flow clearer.